packages/discordx/src/Client.ts

```diff
@@ -636,38 +638,33 @@ export class Client extends ClientJS {
       this.logger.info(str);
     }
 
```

Comment:
Not sure how to as the typings don't properly match up since guild `delete`s are slightly different as they are differently typed. Feel free to suggest.